scripts/preprocessor.py

```diff
@@ -8,7 +8,7 @@
 from hashlib import sha256
 from utils import *
 
-attributes_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../smpc-global/', 'attributes.json')
+attributes_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../smpc-global-master/', 'attributes.json')
```

Comment:
The repository name is `smpc-global` and it is a submodule inside `smpc-local-player`